utils/vscode/package.json

```diff
@@ -0,0 +1,32 @@
+{
+    "name": "spirv",
+    "description": "Language support for SPIR-V disassembly files",
+    "author": "Google",
+    "license": "Apache-2.0",
+    "version": "0.0.1",
+    "private": true,
+    "publisher": "Google",
+    "engines": {
+        "vscode": "^0.10.10"
+    },
+    "categories": [
+        "Programming Languages"
+    ],
+    "contributes": {
+        "languages": [
+            {
+                "id": "spirv",
+                "extensions": [
+                    "spirv"
+                ]
+            }
+        ],
+        "grammars": [
+            {
+                "language": "spirv",
+                "scopeName": "source.spirv",
+                "path": "spirv.json"
+            }
+        ]
+    }
+}
```

Comment:
Could add new line :)

Reply:
Done.
Also updated with proper support for multi-line strings, tokenization of identifiers. I've also reshuffled the code a little.